packages/math-input/src/components/echo-manager.tsx

```diff
@@ -19,6 +19,8 @@ type EchoProps = {
     borders: Border;
     id: Key;
     initialBounds: Bound;
+    heightPx: number;
+    widthPx: number;
```

Comment:
**style nit:** we don't normally put suffix on `height` and `width`.  The assumption that React uses is that if the value doesn't contain a unit of measure that the value is assumed to be in pixels.